scripts/deploy.sh

```
@@ -1,96 +1,45 @@
-if [ $CURRENT_PROFILE == set1 ]
-then
-  IDLE_PROFILE=set2
-  IDLE_PORT=8082
-elif [ $CURRENT_PROFILE == set2 ]
-then
-  IDLE_PROFILE=set1
-  IDLE_PORT=8081
+set -e
+
+APP_DIR=/home/ec2-user/app
+JAR_PATH=$(ls $APP_DIR/*.jar | grep -v plain | head -1)
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)